tests/test_vision_explanation.py

```diff
@@ -8,6 +8,7 @@
 import urllib.request as request_file
 
 import matplotlib.pyplot as plt
+import torch
```

Comment:
Consider trapping import error or perhaps updating README that torch is not installed from dependencies file and should be installed from pytorch's official website - https://pytorch.org/get-started/locally/

Reply:
trapped torch and vision explanation methods